k8s/operators/src/pool/diskpool/client.rs

```diff
@@ -178,6 +178,41 @@ pub(crate) async fn migrate_to_v1beta1(
     Ok(())
 }
 
+/// List pool from control plane. Checks if DSP cr present for it. If no, create it ourself.
```

Comment:
Something like this:

```suggestion
/// Reconciles control-plane pools into CRs by listing pools from the control plane and creating equivalent CRs if respective CR is not present.
```